docs/chapter_phil/What is probability? Bayes contra Hume.rst

```diff
@@ -1,3 +1,801 @@
+it the problem that we have a possible epistemic (belief or subjective) and an 
+empirical (frequencies or objective) interpretation of it.
+
+For some time having two sides of a coin was not that much of a problem and 
+both possible interpretations coexisted. You will find all kinds of important 
+work on probability from all the names you can find in a typical textbook on 
+statistics – from Huygens to the Bernoullli family, Poisson, Gauss, Borel, 
+Markov to Kolmogorov who in 1933 showed that the mathematical results of 
+probability follow a basic set of axioms (`Dienes, 2008`_; `Galavotti, 2015`_). 
+But a few names stand out in this history of probability – Bayes, Laplace, 
+Fisher, Neyman and Pearson - because they are basic to our understanding of 
+probability in statistics and its opposing school, so let´s zoom in on them.
+
+Thomas Bayes (1702-1761) was a minister and a member of the Royal Society with 
+only one publication under his name during his lifetime. He became the name 
+giver of the ‘Bayesian school’ or the subjective probability when his 
+friend Richard Price published a manuscript of his after his death. The 
+manuscript was on the problem of how one may obtain the probability of a 
+hypothesis given some data – something we would later call the Bayes’ 
+theorem. This 
+concept was as simple as it was beautiful – We modify our opinion with 
+objective information: Initial beliefs + recent objective data = new and 
+improved belief (`McGrayne, 2011`_). We will have a closer look at this theorem 
+later in this chapter.
+
+Bayes’ ideas were picked up by Pierre-Simon Laplace (1749-1827) in the early 
+19th century. Laplace was a probabilist and even referred to as the “Newton 
+of France” for his work in physics and astronomy. In his lifetime he came up 
+with two theories of probability that showed more or less the same results with 
+very large samples (`Galavotti, 2015`_). Just on a quick side note – Laplace 
+despised gambling.
+
+Times changed; politics as well as personal animosities played their part as 
+much as a shift in the idea what ‘science’ could do, and with it crashed 
+the idea that there is a balance between subjective beliefs and objective 
+frequencies. The new natural science wanted a ‘route to certain knowledge’, 
+and how could this be subjective in any way? This led to scientists treating 
+the two points of view in probability as diametrical (`McGrayne, 2011`_). 
+Soon mathematicians should take over the field of probability, bringing with 
+them the frequentist approach or objective probability and most terms, concepts 
+and techniques we use in statistics today (`Dienes, 2008`_).
+
+The most dominant name in this upcoming approach would become Sir Ronald Fisher 
+(1890-1962), a geneticist, who coined the terms ‘null hypothesis’ and 
+‘significance’, and suggested a p-value of .05 as an arbitrary level to judge a 
+result statistically significant (`Dienes, 2008`_). After working for years on 
```

Comment:
I would hedge this - Fisher is perhaps directly responsible for the overabundant $p < .05$ standard, but it's not like he said, "a result is publishable if it meets the .05 p value, final word and law forever!". Rather he suggested it in *some* of his writings as a reasonable standard for considering that further investigating a phenomenon.

Reply:
Maybe rephrase to: 

>  and significance (Dienes, 2008). Though often deemed the originator of the tradition of using p < .05 as an arbitrary level to judge results as statistically significant, Fisher (1926, p. 85) was open in suggesting that each researcher should select a threshold in relation to the specific study. As a matter of fact, Gigerenzer, Krauss, and Vitouch (2004, p. 392) suggest that "Fisher thought that using a routine 5% level of significance indicated lack of statistical thinking".